modules/querier/http.go

```diff
@@ -50,6 +57,21 @@ func (q *Querier) TraceByIDHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.Header.Get("Accepts") == "application/protobuf" {
+		level.Info(cortex_util.Logger).Log("msg", "received content type application/grpc")
+		trace := &tempopb.Trace{Batches: resp.Trace.Batches}
```

Comment:
is there a need to create a new trace here?  `resp.Trace` should be marshallable directly?

Reply:
I swear I was seeing some issues with it being a pointer, but you're right, its marshallable directly.